src/SFA.DAS.Commitments.Api/Web.config

```diff
@@ -173,7 +173,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.1.1.2" />
```

Comment:
This _may_ break Commitments Api - is it required for this change, and if not can this change be removed?

Reply:
done